web/components/answers/answers-panel.tsx

```diff
@@ -497,6 +522,27 @@ function Answer(props: {
                   Edit
                 </Button>
               )}
+
+            {unfilledBets?.length && limitOrderVolume ? (
```

Comment:
If length is 0, won't it render '0' instead of nothing? Classic React.

Reply:
It shouldn't because its a ternary right? If its 0 it will fallback to `null`

Reply:
Oh my bad, you're right, didn't see the ternary